test/corpus/regression.txt

```diff
@@ -12,12 +12,12 @@ Comments within a list
 ---
 
 (program
-  (function_call
-    name: (identifier)
+  (list
```

Comment:
Note that while this is correct. In fennel it's only a "list" when it's within a macro, outside of a macro it's considered a function call

Reply:
You cannot distinguish them though.